ENCRYPTION.md

```diff
@@ -39,7 +45,7 @@ needs specific parameters to get a key.
 
 ## Installing and Configuring the Encryption Plug-In
 
-1. Configure the sharedLib directory in solr.xml (e.g. sharedLIb=lib) and place the Encryption plug-in jar file into
+1. Configure the sharedLib directory in solr.xml (e.g. sharedLib=lib) and place the Encryption plug-in jar file into
```

Comment:
Preferably we simply refer to Solr's Ref Guide, which enumerates some options: https://solr.apache.org/guide/solr/latest/configuration-guide/libs.html